apps/web/app/types/network.ts

```diff
@@ -0,0 +1,21 @@
+import { ChainId, StacksNetworkName } from '@stacks/network';
+
+export interface Network {
+  label: string;
+  url: string;
+  networkId: ChainId;
+  mode: StacksNetworkName;
+  wsUrl?: string;
+  isCustomNetwork?: boolean;
+}
+
+interface WhenStacksModeMap<T> {
+  mainnet: T;
+  testnet: T;
+  devnet: T;
+  mocknet: T;
+}
```

Comment:
Stacks only has two "modes" in the way were using mode for bitcoin. These are networks so `WhenStacksNetworkMap` would be better